Client/src/Components/DataCell/DataCell.js

```diff
@@ -25,24 +42,47 @@ export default function DataCell() {
             method: 'POST',
             body: formData,
         };
-        await fetch('http://localhost:4000/dataupload', options)
-            .then(resp => resp.json())
-            .then(result => {
-                console.log(result[0])
-            })
-    }
 
+        try {
+            await fetch('http://localhost:4000/dataupload', options)
+                .then(resp => resp.json())
+                .then(result => {
+                    console.log(result[0])
+                })
+        } catch (err) {
+            console.log('wrong file submitted, only json file accepted')
+        }
+    }
+    const handleClose = (event, reason) => {
+        if (reason === 'clickaway') {
+            return;
+        }
+        setOpen(false);
+    };
+    //Function to get the extension of a file
+    function getExtension(filename) {
+        var i = filename.lastIndexOf('.');
+        return (i < 0) ? '' : filename.substr(i);
+    }
+    //Snackbar is used to show an error on the screen when a wrong file type is selected for uploading
     return (
-        <Container>
-            <Box border={30} p={4} borderColor="primary">
-                <form onSubmit={handleSubmit}>
-                    <img src={require('./uploadimage.png')} alt="Visual of clouds"></img>
-                    <div>
-                        <input type="file" id="jsonFile" accept=".json" />
-                        <Button type="submit" variant="contained"> Upload this file! </Button>
-                    </div>
-                </form>
-            </Box>
-        </Container>
+        <>
+            <Snackbar open={open} autoHideDuration={3000} onClose={handleClose}>
+                <Alert onClose={handleClose} severity="error">
+                    Failed to parse json
+                </Alert>
+            </Snackbar>
+            <Container>
+                <Box border={30} p={4} borderColor="primary">
+                    <form onSubmit={handleSubmit}>
+                        <img src={require('./uploadimage.png')} alt="Visual of clouds"></img>
+                        <div>
+                            <input type="file" id="jsonFile" accept="application/json" />
```

Comment:
You will get merge conflict here for id= - lets keep the "file" one currently in develop

Reply:
I have just fixed all merge conflicts with develop